recipe/meta.yaml

```diff
@@ -23,11 +23,11 @@ requirements:
     - pip
     - python >=3.6
     - setuptools
-    - numpy
+    - numpy >=1.16.0
```

Comment:
Same comment here as in nexpy.

Reply:
As with the NeXpy update, I have now removed it from the host requirements.